tests/functional/shock/shock.py

```
@@ -149,7 +149,10 @@ def main():
                     interp_order, interp_order, interp_order
                 )
             )
-            subprocess.call(cmd)
+            try:
+                subprocess.call(cmd)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)